### PR TITLE
[MBO-312] Get user menu from API

### PR DESCRIPTION
**Pull request by intraordinaire**

### CODE REVIEW

**Comment by intraordinaire** [comment]
Waiting for the Nest part to me merged

File: src/Distribution/Client.php [hunk 1]
```diff
@@ -157,6 +169,38 @@ public function registerShop(string $token): stdClass
...
+     * Retrieve the user menu from NEST Api
+     *
+     * @return \stdClass
+     *
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getConf(): stdClass
+    {
+        $language = $this->contextBuilder->getLanguage();
+        $cacheKey = __METHOD__ . $language->getLanguageCode() . _PS_VERSION_;
```
Comment (sowbiba) [inline src/Distribution/Client.php:176]: you can put `$language->getLanguageCode()` in a variable as it is reused later

File: src/Service/View/ContextBuilder.php [hunk 2]
```diff
@@ -141,7 +141,7 @@ private function getContext(): Context
         return $this->contextAdapter->getContext();
     }
 
-    private function getLanguage(): Language
+    public function getLanguage(): Language
```
Comment (sowbiba) [inline src/Service/View/ContextBuilder.php:144]:
```suggestion
    private function getLanguage(): Language
```

**Review by sowbiba — COMMENTED** [review]
Nice work !
just some details ...

**Comment by intraordinaire** [comment]
Testable with the prestabulle3

**Review by sowbiba — APPROVED** [review]
Thanks @intraordinaire

**Pull request merged**